code/game/objects/items/crab17.dm

```diff
@@ -5,23 +5,26 @@
 	icon_state = "suspiciousphone"
 	w_class = WEIGHT_CLASS_SMALL
 	attack_verb = list("dumped")
-	var/dumped = FALSE
+	COOLDOWN_DECLARE(crab_reactivation_cooldown)
 
 /obj/item/suspiciousphone/attack_self(mob/user)
 	if(!ishuman(user))
 		to_chat(user, "<span class='warning'>This device is too advanced for you!</span>")
 		return
-	if(dumped)
-		to_chat(user, "<span class='warning'>You already activated Protocol CRAB-17.</span>")
+	if(!COOLDOWN_FINISHED(src, crab_reactivation_cooldown))
+		to_chat(user, "<span class='warning'>CRAB-17 will be reactivated in [round(COOLDOWN_TIMELEFT(src, crab_reactivation_cooldown)/10)] seconds.</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>CRAB-17 can be reactivated in [round(COOLDOWN_TIMELEFT(src, crab_reactivation_cooldown)/10)] seconds.</span>")
```

Will implies that it's queueing usage, as opposed to informing you of a cooldown.